modules/cnv-about-fencing.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-fencing-nodes.adoc
+
+[id="cnv-about-fencing_{context}"]
+
+= About fencing nodes
+
+Fencing is the act of protecting shared resources in order to safely recover 
+from node failure.
+
+In the case of an unhealthy node, Pods and virtual machines that depend on 
+persistent storage cannot be replaced on another node until it can be confirmed 
+that the original Pod or virtual machine is no longer accessing the storage. 
```

Comment:
If we dropped the reference to virtual machines, the remainder would be completely appropriate to include in the main OCP docs.  Maybe we should do that?

Reply:
Completely agree. Once we've nailed down the content we can use conditionals to only include references to virtual resources when used in a CNV context.